src/java/TypesUtils.cpp

```diff
@@ -342,6 +342,7 @@ static bool isJavaMap(JNIEnv * env, jobject obj)
 
 static Object * hashmapJavaToObjectConverter(JNIEnv * env, jobject obj)
 {
+	env->PushLocalFrame(20);
```

Comment:
I see now there are other places in TypesUtils.cpp that already call PushLocalFrame with a value of 32. I will create a constant and make sure all calls use that constant